CMakeLists.txt

```diff
@@ -140,7 +140,11 @@ endif()
 if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
 	list(APPEND yquake2LinkerFlags "-lm -static-libgcc")
 else()
-	list(APPEND yquake2LinkerFlags "-lm -rdynamic")
+	if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Haiku")
+		list(APPEND yquake2LinkerFlags "-lm -rdynamic")
+	else()
+		list(APPEND yquake2LinkerFlags "-lm -lnetwork")
+	endif()
```

Comment:
Do we need the same change for the Makefile?

Reply:
You re right I forgot